Makefile

```diff
@@ -212,6 +212,9 @@ else
 HELM_DOCS=$(shell which helm-docs)
 endif
 
```

Comment:
@claudiolor Answering here to your previous comment in the main thread: it could be nice adding a comment here such as:
#generate .md file about liqoctl, to be included in the Liqo documentation website

Reply:
fixed :)